webpack.config.js

```diff
@@ -35,6 +35,27 @@ const sharedConfig = {
     filename: '[name].js',
     chunkFilename: '[name].js',
   },
+  module: {
+    ...defaultConfig.module,
+    rules: [
+      ...defaultConfig.module.rules,
+      {
+        test: /\.svg$/,
+        use: ['@svgr/webpack', 'url-loader'],
+      },
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader', 'postcss-loader'],
```

Comment:
In https://github.com/google/web-stories-wp/pull/351 'postcss-loader' is removed.

Reply:
Just moving these lines around here, when either PR is merged there will be a conflict that needs to be resolved.